### PR TITLE
Updated documentation

### LINKED ISSUE #172: As a user, I want to know how to upload a new profile picture via Gravatar

### LINKED ISSUE #177: Write user documentation of the new features.
Update the user documentation.

### PR DESCRIPTION
**Pull request by juanecabellob**
Closes #177, Closes #172

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -159,8 +159,11 @@ def user_create_confirm(request, token):
 
 def user_login(request):
     if request.method == "GET":
-        template = loader.get_template('users/user_login.html')
-        return HttpResponse(template.render(request=request))
+        if not request.user.is_authenticated():
+            template = loader.get_template('users/user_login.html')
+            return HttpResponse(template.render(request=request))
+        else:
+            return redirect('users:profile', pk=request.user.pk)
```
Comment (Gjum) [inline users/views.py]: Does this belong into this PR?

Comment (Gjum) [inline users/views.py]: Rebase please.

**Comment by juanecabellob** [comment]
Closes #177, Closes #172

File: MANUAL.md [hunk 2]
```diff
@@ -1,12 +1,35 @@
 # Welcome to the Circuit user manual
```
Comment (Gjum) [inline MANUAL.md:1]: ~~@juanecabellob Could you link to this headline directly? like this: [MANUAL.md#welcome-to-the-circuit-user-manual](#welcome-to-the-circuit-user-manual)~~ Done, thanks.

File: MANUAL.md [hunk 3]
```diff
@@ -59,52 +84,214 @@ support@circuit.io
...
+
+Jelisa [@Iotka](http://github.com/Iotka)
+
+Nico [@nrigheriu](http://github.com/nrigheriu)
+
+
+## Troubleshoot
+
+### Bugs?
+We are humans! It can come to times where our circuits get mixed and you see the resulting short circuit.
```
Comment (metalmatze) [inline MANUAL.md:257]: I see what you did there `our circuits get mixed` :smile:

Comment (Gjum) [inline MANUAL.md:257]: I don't get it.

**Pull request merged**